<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PChatOptionsWidget</class>
 <widget class="QWidget" name="PChatOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>164</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PChatOptionsWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="_LabelLayout">
     <item>
      <widget class="QLabel" name="_LabelLabel">
       <property name="text">
        <string>Label: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_LabelEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_ShowLayout">
     <item>
      <widget class="QCheckBox" name="_ShowCheck">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_DockCheck">
       <property name="text">
        <string>Dock</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="_ChannelBox">
     <property name="title">
      <string>Channels</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QCheckBox" name="_LocalCheck">
        <property name="text">
         <string>Local</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="_PartyCheck">
        <property name="text">
         <string>Party</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="_WhisperCheck">
        <property name="text">
         <string>Whisper</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="_TradeCheck">
        <property name="text">
         <string>Trade</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="_GlobalCheck">
        <property name="text">
         <string>Global</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="_GuildCheck">
        <property name="text">
         <string>Guild</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PoePal.qrc"/>
 </resources>
 <connections/>
</ui>
